packages/meditrak-server/src/externalApiSync/SyncQueueChangesManipulator.js

```diff
@@ -25,7 +25,14 @@ export class SyncQueueChangesManipulator {
   getIdsFromChangesForModel = (changes, model) =>
     this.getRecordIds(this.getChangesForRecordType(changes, model.databaseType));
 
-  getRecords = changes => changes.map(c => c.record);
+  getRecords = changes =>
+    changes.map(change => {
+      const { old_record: oldRecord, new_record: newRecord, type } = change;
+      if (type === DELETE) {
+        return oldRecord;
+      }
+      return newRecord;
```

Comment:
Haha I'm happy to go with it - though I also don't mind some wasted lines for clarity ;-)

Reply:
Agree, my main drive is not line count but easier reading! Of course, this is subjective

Reply:
Yeah I think we disagree on which is easier reading, but I am only mildly in favour of the `if`, and have used the ternary form many other times. I'm going with that because I believe it's the more common in our codebase.

Reply:
Good to know, I thought that people often don't use `? :` because of how their code involves, not as a conscious readability choice. It is a matter of taste then, feel free to use whatever you prefer and to ignore the change suggested above